src/Zicht/Bundle/SolrBundle/Manager/SolrManager.php

```diff
@@ -41,19 +42,28 @@ class SolrManager
     public function __construct(Client $client)
     {
         $this->client = $client;
-        $this->mappers = array();
+        $this->mappers = [];
     }
 
 
     /**
      * Adds a data mapper
      *
      * @param DataMapperInterface $dataMapper
+     *
      * @return void
      */
     public function addMapper($dataMapper)
     {
-        $this->mappers[]= $dataMapper;
+        $this->mappers[] = $dataMapper;
+    }
+
+    /**
+     * @return DataMapperInterface[]
+     */
+    public function getMappers()
```

Comment:
This method is in here twice all of a sudden :see_no_evil:. Let me fix that :grin: